documentation/modules/deploying/proc-deploy-migrate-kraft.adoc

```diff
@@ -23,7 +23,7 @@ Note also, the following:
 
 * You must be using Strimzi 0.40 or newer with Kafka 3.7.0 or newer. If you are using an earlier version of Strimzi or Apache Kafka, upgrade before migrating to KRaft mode.
 * Verify that the ZooKeeper-based deployment is operating without the following, as they are not supported in KRaft mode:
-** The Topic Operator running in bidirectional mode. It should either be in unidirectional mode or disabled.
+** The Topic Operator running in bidirectional mode. It should either be in unidirectional mode (default since Strimzi 0.39) or disabled.
```

Comment:
I think you can simply remove this, or? In 0.41, it will be always either disabled or running as UTO.